### PR TITLE
DOCS-11868 adding content to modify email alerts

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/configure-alerting.adoc [hunk 1]
```diff
@@ -5,12 +5,34 @@ endif::[]
...
+| `kubernetes_node` | Sends a warning when more than 90% of CPU is used. Triggers a critical error when more than 95% is used.
+| `muleapps` | Triggers a critical error when more than 90% of CPU is used. 
+| `network_params` | Triggers a critical error if networking is disabled on a node. 
+| `uptime` | Sends a warning when the system is inaccessible for longer than 5 min.
+|===
 
-== Procedure
+== Configure Alerts
 
+You can send Runtime Fabric alerts from a designated alert sender email address through your SMTP server. You can configure only one alerts email recipient.
```
Comment (valkyrie69) [inline modules/ROOT/pages/configure-alerting.adoc]: What about: You can configure only one email recipient for alerts.

Comment (valkyrie69) [reply modules/ROOT/pages/configure-alerting.adoc]: one alerts email... sounds wrong but I don't think it is technically.

File: modules/ROOT/pages/configure-alerting.adoc [hunk 2]
```diff
@@ -59,31 +81,69 @@ $ sudo gravity resource create -f alert-smtp.yaml
...
-Currently, only one alerts email recipient is supported.
+== Modify Email Alert Contents
 
-== Built-in Alerts
+If needed, you can modify the content of email alerts sent from Runtime Fabric. Alerts are written in TICKscript.
 
-The following table lists default alerts provided by Anypoint Runtime Fabric on VMs / Bare Metal:
+[NOTE]
+--
+If you customize any Runtime Fabric email alerts, appliance upgrades overwrites the customizations.
```
Comment (valkyrie69) [inline modules/ROOT/pages/configure-alerting.adoc]:
```suggestion
If you customize any Runtime Fabric email alerts, appliance upgrades overwrite the customizations. 
```

File: modules/ROOT/pages/configure-alerting.adoc [hunk 3]
```diff
@@ -59,31 +81,69 @@ $ sudo gravity resource create -f alert-smtp.yaml
...
++
+----
+kubectl -n monitoring exec <pod name> -c alert-loader -- kapacitor -url http://localhost:9092 list tasks
+----
+. Delete the alert you modified:
++
+----
+kubectl -n monitoring exec <POD NAME> -c alert-loader -- kapacitor -url http://localhost:9092 delete tasks <MODIFIED ALERT NAME>
+----
+. Check that the alert is recreated and the status/executing values are enabled/true:
```
Comment (valkyrie69) [inline modules/ROOT/pages/configure-alerting.adoc]:
```suggestion
. Verify that the alert is re-created, the Status value is `enabled`, and the Executing value is `true`:
```

File: modules/ROOT/pages/configure-alerting.adoc (hunk 1 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/configure-alerting.adoc]:
```suggestion
You can send Runtime Fabric alerts from a designated alert sender email address through your SMTP server. You can configure only one email recipient for alerts.
```

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**